xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-livetable/src/main/java/org/xwiki/livedata/internal/livetable/LiveTableLiveDataPropertyStore.java

```diff
@@ -155,6 +154,7 @@ private LiveDataPropertyDescriptor getLiveDataPropertyDescriptor(PropertyClass x
                 // The default live table results page currently supports only exact matching for list properties with
                 // multiple selection and no relational storage (selected values are stored concatenated on a single
                 // database column).
+                descriptor.getFilter().addOperator("empty", null);
```

Comment:
I take it that the default ``LiveTableResults`` page supports empty list filter only for ``StringListProperty``? What operators are available for ``DBStringListProperty`` (relational storage)?

Reply:
> I take it that the default LiveTableResults page supports empty list filter only for StringListProperty?
No, that's not really what this change means. 

Previously to my change the `empty` operator was added to any descriptor with type list, and when we're getting the default operator we're checking first the operators defined in the descriptor: *this list is not merged*. Which means that in case of a list which wasn't a StringList we were always getting only `empty`... 

Now for a  `StringListProperty` we do want to override the list of operators to only use equals because others are not supported (according to the comment), but we still also need to support empty, so I just moved it there. 
The result is that for all list properties there no operator defined in the descriptor: we entirely rely on the type filters which are defining all the operators (including empty). And for `StringListProperty` we rely on the descriptor which defines the two operators `equals` and `empty`.

Now when writing this I'm still wondering if there's not a hidden bug here with `StringListProperty` because 1. the operators defined in the descriptor are only used in the column filter, not in the custom filters from the panels (which are always coming directly from the type) 2. I have a doubt on the code of applyFilter here: I think it would default on the descriptor operator first, but we probably need to also define the default operator in the descriptor, to ensure it doesn't fallback on empty operator, but on equals. @manuelleduc wdyt?